.gitignore

```diff
@@ -53,3 +53,5 @@ doc/**
 
 # secrets
 tests/sensu_license.json
+tests/ci_build.sh
```

Comment:
is this being auto generated? where?

Reply:
It's in `tests/secrets.tar`. It has the secret token for packagecloud so only exists inside encrypted tar archive. We can only encrypt one file and now require two files be encrypted and travis docs say in that case to encrypt a tar archive with files needing to be kept secret.

Reply:
Could you add instructions on how this file would be decrypted for local development

Reply:
I added scripts and added those scripts to `tests/README.md`. The decryption of those files is only necessary if one wishes to run tests against a CI build.  The absence of the secrets being decrypted and certain env variables will just test latest production package.